public/personality.html

```diff
@@ -831,6 +831,7 @@ <h2 class="personality__title">Pop the balloons <br> that don't <br>describe you
     </div>
   </div>
   <script src="balloon.js"></script>
+  <script src="add-to-storage.js"></script>
   <script src="swipe.js"></script>
```

Comment:
It feels a bit like you just make a new file whenever you want to build a new function.. This is really not a good way of going forward. pls address https://github.com/CYPIAPT-LNDSE/welcome-to-camhs/issues/58